src/components/darien/DarienHome.jsx

```diff
@@ -17,47 +19,54 @@ import {
   PROGRAMS_INITIAL_STATE, PROGRAMS_PROPTYPES, PROGRAMS_STATUS
 } from '../../ducks/programs';
 
-class DarienHome extends React.Component {
-  constructor(props) {
-    super(props);
-  }
+const DarienHome = (props) => {
+  const signedIn = (props.user && props.initialised);
+  const selectedProgramExists = (props.programsStatus === PROGRAMS_STATUS.SUCCESS && props.selectedProgram);
+  const name = (selectedProgramExists && props.selectedProgram.name) ? props.selectedProgram.name : '';
 
-  render() {
-    const selectedProgramExists = (this.props.programsStatus === PROGRAMS_STATUS.SUCCESS && this.props.selectedProgram);
-    const name = (selectedProgramExists && this.props.selectedProgram.name) ? this.props.selectedProgram.name : '';
-
-    return (
-      <ProgramHome className="darien-home">
-        <Hero
-          className="program-home__hero"
-          background={<Image src="https://placeimg.com/1000/1000/nature/any" fit="cover" />}
-          backgroundColorIndex="dark"
-          size="medium"
-        >
-          <Box align="center"><Heading className="program-home__header">{name}</Heading></Box>
-        </Hero>
-        <Section
-          className="program-home__section"
-          align="center"
-          colorIndex="accent-3"
-          direction="column"
-          margin={{ vertical: 'none', horizontal: 'none' }}
-          pad={{ vertical: 'none', horizontal: 'none' }}
-          justify="center"
-        >
-          <Box align="center" direction="column">
-            <Paragraph>Investigate ecological questions by exploring trail camera data using an interactive map. Filter and download data to perform analyses and test hypotheses. If you are an educator, you can set up private classrooms and invite your students to join. Curate data sets or let your students explore on their own. Guided activities and supporting educational resources are also available. If you are a student or you simply want to explore the data, click the Explorer button.</Paragraph>
-            <Paragraph>Are you an educator or a student/explorer? Make your selection to get started!</Paragraph>
+  return (
+    <ProgramHome className="darien-home">
+      <Hero
+        className="program-home__hero"
+        background={<Image src="https://placeimg.com/1000/1000/nature/any" fit="cover" />}
```

Comment:
The Darien team might want to pick an actual background image here rather than use this placeholder. Maybe something from the project? 

I put the string for the background source into the program metadata for I2A just so it's consistent with referencing everything about the program from the database. Just so you know, the controllers for update hasn't been done for programs on the API, so you have to ask for someone update it via the Rails console.